<?xml version="1.0"?>
<launch>
    <arg name="mav_name"        default="quadrotor"/>

    <!-- <include file="$(find iarc_simulation_tools)/launch/default.launch">
        <arg name="mav_name"    value="$(arg mav_name)"/>
    </include> -->

    
    <group ns="$(arg mav_name)">
        <node pkg="pose_estimator"  type="pose_estimator_node"  name="pose_estimator_node"  output="screen">
            <rosparam file="$(find pose_estimator)/config/param.yaml"/>
            <remap from="odom"          to="ground_truth/odometry"/>
            <remap from="centre_coord"  to="detector/centre_coord"/>
        </node>
        <!-- <node pkg="box_detector" type="box_detector" name="box_detector" output="screen">

        </node> -->
    </group>

</launch>
